cobalt/media/media_module.cc

```diff
@@ -252,6 +266,31 @@ bool MediaModule::SetConfiguration(const std::string& name, int32 value) {
   return false;
 }
 
+void MediaModule::SetPreferMinimalPostProcessingFromPersistentSettings() {
+  // Called on initialization and when the persistent setting is changed.
+  if (options_.persistent_settings == nullptr) {
+    return;
+  }
+
+  base::Value value;
+  options_.persistent_settings->Get(
+      kPreferMinimalPostProcessingPersistentSettingsKey, &value);
+  bool prefer_minimal_post_processing = value.GetIfBool().value_or(false);
+
+  const StarboardExtensionMediaSettingsApi* media_settings_api =
+      static_cast<const StarboardExtensionMediaSettingsApi*>(
+          SbSystemGetExtension(kStarboardExtensionMediaSettingsName));
+  if (media_settings_api &&
+      strcmp(media_settings_api->name, kStarboardExtensionMediaSettingsName) ==
+          0 &&
+      media_settings_api->version >= 3) {
+    media_settings_api->SetPreferMinimalPostProcessing(
+        prefer_minimal_post_processing);
+    LOG(INFO) << "Set PreferMinimalPostProcessing to "
```

Comment:
Is this really useful? I'm always wary about logspam.

Reply:
Removed.